feedback_tool.py

```diff
@@ -0,0 +1,71 @@
+import psycopg2
+import random
+import streamlit as st
+from transformers import BartForConditionalGeneration, BartTokenizer
+
+# establishing connection to the database used
+my_connection = psycopg2.connect(database="postgres", user="souadyakubu", password="souadseny", host="localhost", port="5432")
+cursor = my_connection.cursor()
```

Comment:
I'm unsure it's a good idea to have a cursor globally. I think you can do `with connection: connection.execute(...)` see https://docs.python.org/3/library/sqlite3.html#sqlite3-connection-context-manager